VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.scss

```diff
@@ -38,3 +38,12 @@
   max-height: 85vh;
   z-index: 100;
 }
+
+.vh-btn-text {
+}
+
+@media screen and (max-width: 1024px) {
```

Comment:
I think you need to use screen only to make sure normal devices aren't affected by this change. This page has a list of combinations that affect tablets only. I hope this helps.
[css device queries](https://css-tricks.com/snippets/css/media-queries-for-standard-devices/#ipad-queries)

Reply:
I added 'only'
The fix should be ok for normal devices as it had not the correct layout if you resize the view, now it's should be better.